tx/tx.go

```diff
@@ -33,15 +33,15 @@ type txData struct {
 	Signature *crypto.Signature
 }
 
-func (tx *Tx) Version() int                 { return tx.data.Version }
-func (tx *Tx) Stamp() crypto.Hash           { return tx.data.Stamp }
-func (tx *Tx) Sequence() int                { return tx.data.Sequence }
-func (tx *Tx) PayloadType() payload.Type    { return tx.data.Type }
-func (tx *Tx) Payload() payload.Payload     { return tx.data.Payload }
-func (tx *Tx) Fee() int64                   { return tx.data.Fee }
-func (tx *Tx) Memo() string                 { return tx.data.Memo }
-func (tx *Tx) PublicKey() *crypto.PublicKey { return tx.data.PublicKey }
-func (tx *Tx) Signature() *crypto.Signature { return tx.data.Signature }
+func (tx *Tx) Version() int                 { return tx.data.Version }   // Transaction version
+func (tx *Tx) Stamp() crypto.Hash           { return tx.data.Stamp }     // hash of previus block
```

Comment:
I think these comments are not necessary. Code itself should talk. 
Here  some comments are not accurate like stamp or sequence. Let's keep the code uncommented.